liota/lib/utilities/utility.py

```diff
@@ -123,6 +124,21 @@ def mkdir_log(path):
             else:
                 raise
 
+
+def ordered_list(obj):
```

Comment:
Why should we need it in liota/utility ?  We can move it when it is needed.

I meant to have it in `test_aws_iot.py` module itself, but not as a method in `AWSIoTTest` class.  This is because `AWSIoTTest` extends `unittest.TestCase` and all methods within that class will become a test case.

Also please change method name to `validate_json`